docs/configure/styling-and-css.md

```diff
@@ -2,7 +2,7 @@
 title: 'Styling and CSS'
 ---
 
-There are many ways to include CSS in a web application, and correspondingly there are many ways to include CSS in Storybook. Usually, it is best to try and replicate what your application does with styling in Storybook’s configuration. To make this easier, we recommend using [`@storybook/addon-styling`](https://github.com/storybookjs/addon-styling).
+There are many ways to include CSS in a web application, and correspondingly there are many ways to include CSS in Storybook. Usually, it is best to try and replicate what your application does with styling in Storybook’s configuration. To make this easier, we recommend using [`@storybook/addon-styling-webpack`](https://storybook.js.org/addons/@storybook/addon-styling-webpack/).
```

Comment:
Worth calling out the type of projects this should (or should not) be used with?

Reply:
Great call out! I've added a few lines about this.